clang/test/Driver/sycl-linker-wrapper-image.cpp

```diff
@@ -43,12 +43,12 @@ int main() {
 // CHECK-DAG: @SYCL_PropSetName.3 = internal unnamed_addr constant [25 x i8] c"SYCL/device requirements\00"
 // CHECK-DAG: @SYCL_PropSetName.4 = internal unnamed_addr constant [22 x i8] c"SYCL/kernel param opt\00"
 // CHECK-DAG: @__sycl_offload_prop_sets_arr.5 = internal constant [3 x %_pi_device_binary_property_set_struct] [%_pi_device_binary_property_set_struct { ptr @SYCL_PropSetName, ptr @__sycl_offload_prop_sets_arr, ptr getelementptr inbounds ([1 x %_pi_device_binary_property_struct], ptr @__sycl_offload_prop_sets_arr, i64 1, i64 0) }, %_pi_device_binary_property_set_struct { ptr @SYCL_PropSetName.3, ptr @__sycl_offload_prop_sets_arr.2, ptr getelementptr inbounds ([1 x %_pi_device_binary_property_struct], ptr @__sycl_offload_prop_sets_arr.2, i64 1, i64 0) }, %_pi_device_binary_property_set_struct { ptr @SYCL_PropSetName.4, ptr null, ptr null }]
-// CHECK-DAG: @.sycl_offloading.0.data = internal unnamed_addr constant [740 x i8] 
+// CHECK-DAG: @.sycl_offloading.0.data = internal unnamed_addr constant [772 x i8] 
```

Comment:
any idea what causes the changes here?

Reply:
This part of testing is evolving. As we continue to add more features to SYCL offloading support in the new offload model, we update the list of options being passed to the link/post-link tools. This in turn impacts code generation. I am not sure if we are at a stage where we can accurately specify what the wrapped image should contain. But, I would let the test exist and update it for correctness as our implementation gets closer to the final product.

Thanks

Reply:
Sounds good, probably we are passing an argument somewhere we missed before.